docs/migrations/00397.md

```diff
@@ -0,0 +1,8 @@
+#### In `Select` component use `maxDropdownHeight` instead of `rows` to setup max height of Select DropDown
+
+##### `maxDropdownHeight` accepts any height style as a string. For `px` value it can prevent Dropdown from being hidden and always set to no more 90% of screen
```

Comment:
I am not sure if I understand this part - can you rephrase? also may worth mentioning the default values?

Reply:
rephrased